src/styles/app.css

```diff
@@ -36,3 +50,25 @@ h2 a.tutorial-header-link {
 h2:hover a.tutorial-header-link {
   text-decoration: underline;
 }
+
+.qpp-docs-title-container {
+	background: #036;
+	padding-top: 30px;
+	padding-bottom: 10px;
+}
+
+.qpp-docs-title-content-container {
+	margin-left: auto;
+	margin-right: auto;
+	max-width: 1100px;
```

Comment:
Can we dry this up with `.container` above - in other words remove the duplicate attributes use the `.container` class above `<div className="qpp-docs-title-content-container container">`